frontend/src/shared/lib/trimAllObjectValues.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Применяет trim ко всем полям объекта
+ * @param object объект для изменения
+ */
+export const trimAllObjectValues = (object: object) => {
```

Comment:
```suggestion
export const trimAllObjectValues = (object: Record<any, string>) => {
```
Позволит избежать ошибок, еслив  полях будет не текст
Можно не вносить, если будет внесена правки ниже